bundles/ranvier-combat/lib/Combat.js

```diff
@@ -19,7 +19,7 @@ class Combat {
    */
   static updateRound(state, attacker) {
     if (attacker.getAttribute('health') <= 0) {
-      Combat.handleDeath(attacker);
+      Combat.handleDeath(state, attacker);
```

Comment:
Something about having to pass the state around everywhere here feels wrong, I'll take a longer look tomorrow afternoon

Reply:
Yeah, I agree with you, it is strange to pass it everywhere; Let me know if there is a better way to do that